docs/en/observability/logs-ecs-application.asciidoc

```diff
@@ -0,0 +1,206 @@
+[[logs-ecs-application]]
+= ECS formatted application logs
+
+Logs formatted in Elastic Common Schema (ECS) don't require manual parsing, and the configuration can be reused across applications. ECS-formatted logs, when paired with an {apm-agent}, allow you to correlate logs to easily view logs that belong to a particular trace.
+
+You can format your logs in ECS format the following ways:
+
+* <<ecs-loggers,undefined>> plugins for your logging libraries that reformat your logs into ECS format.
```

Comment:
The link text shows up as "undefined." Not sure what text you want to show here, so I won't make a suggestion.